src/components/Form.js

```diff
@@ -178,7 +178,14 @@ const Form = () => {
 
     let submission;
     try {
-      submission = await createSubmission(config.baseUrl, form, config.clientBaseUrl);
+      const anonymous = !!event;
```

Comment:
I'm not sure if there is another way for me to figure out what login option the user selected at this point

Reply:
yeah, in `src/components/LoginOptions/index.js` on line 66 you can provide an explicit argument to indicate that it's an anonymous login:

```jsx
  const containerProps = form.loginRequired
    ? {}
    : {
        onSubmit: e => {
          e.preventDefault();
          onFormStart(e, {anonymous: true});
        },
      };
```
so then just update the `onFormStart` signature to take `(event, {anonymous = false}) {...}`, that makes it explicit :)